dcicutils/docker_utils.py

```diff
@@ -28,6 +30,8 @@ def tag_image(*, image, tag, ecr_repo_name):
         # works by side-effect
         image.tag(ecr_repo_name, tag=tag)
 
-    def push_image(self, tag, ecr_repo_name):
+    def push_image(self, *, tag, ecr_repo_name, auth_config):
         """ Pushes the tag - presumes the above tag_image method has been called. """
-        return self.client.images.push(ecr_repo_name, tag=tag)
+        for line in self.client.images.push(ecr_repo_name, tag=tag, stream=True, decode=True,
+                                            auth_config=auth_config):
+            PRINT(line)
```

Comment:
Will the line be self-identifying (needing no prefix explaining why it's being typed out)?